app/views/cohorts/client_notes/index.haml

```diff
@@ -3,19 +3,24 @@
     = link_to new_cohort_cohort_client_client_note_path(@cohort, @cohort_client), class: 'btn btn-secondary btn-sm', data: {loads_in_pjax_modal: true} do
       %i.icon-plus
       Add a note
-- @cohort_client.client.cohort_notes.order(updated_at: :desc).each do |note|
-  .mb-4
+- if @cohort_client.client.cohort_notes.exists?
+  - @cohort_client.client.cohort_notes.order(updated_at: :desc).each do |note|
```

Comment:
Since you are going to load them all anyway:
```suggestion
- notes = @cohort_client.client.cohort_notes.order(updated_at: :desc).to_a
- if notes.present?
  - notes.each do |note|
```